sdk/python/test/e2e/conftest.py

```diff
@@ -0,0 +1,21 @@
+import os
+import pytest
+
+from kubeflow.training.utils.utils import is_running_in_k8s
+from kubeflow.training import TrainingClient
+
+
+def pytest_addoption(parser):
+    parser.addoption("--namespace", action="store", default="default")
```

Comment:
Should we confirm whether this flag works fine in CI? Maybe we can do that by just adding this flag like the following to https://github.com/kubeflow/training-operator/blob/1cdf4eefe90082b2371ed717103fcb8509896acd/.github/workflows/integration-tests.yaml#L82.

```shell
python3 -m pip install -e sdk/python; pytest sdk/python/test --log-cli-level=info --namespace=e2e
```

Reply:
Tested the above command on my local. Had to move out `confest.py` to the root test directory. It's working fine now.

Reply:
Can we run tests in CI using the below commands to confirm that conftest.py works in every PR? 

```shell
python3 -m pip install -e sdk/python; pytest sdk/python/test --log-cli-level=info --namespace=default
```

This means we add `--namespace=default` to https://github.com/kubeflow/training-operator/blob/1cdf4eefe90082b2371ed717103fcb8509896acd/.github/workflows/integration-tests.yaml#L82.